kotlin-jvm/src/main/kotlin/playground/SkrapeIt.kt

```diff
@@ -0,0 +1,206 @@
+@file:Suppress("PackageDirectoryMismatch")
+
+package playground.skrapeit
+
+import it.skrape.core.htmlDocument
+import it.skrape.expect
+import it.skrape.matchers.*
+import it.skrape.selects.and
+import it.skrape.selects.html5.*
+import it.skrape.skrape
+
+/**
+ *  skrapeit/skrape.it
+ *
+ * [Website](https://docs.skrape.it/docs/)
+ * [GitHub](https://github.com/skrapeit/skrape.it)
+ */
+
+fun main() {
+    println()
+    println("# skrapeit/skrape.it")
+    println(
+        "A Kotlin-based HTML / XML deserialization library that places particular emphasis on ease of use and a " +
+            "high level of readability by providing an intuitive DSL."
+    )
+    println()
+
+    println("Pick Html-Elements from a Doc")
+    `Pick Html-Elements from a Doc`()
+    println("Pick Custom HTML tags")
+    `Pick Custom HTML tags`()
+    println("Build CSS selectors")
+    `Build CSS selectors`()
+    println("Parse and verify local HTML resource")
+    `Parse and verify local HTML resource`()
+    println("Parse and verify response of url")
+    `Parse and verify response of url`()
+}
+
+/**
+ * [Picking Html-Elements from a Doc](https://docs.skrape.it/docs/dsl/parsing-html#picking-html-elements-from-a-doc)
+ */
+private fun `Pick Html-Elements from a Doc`() {
```

Comment:
Can you split this example in two parts?
Both the example and the code that decode it takes so much vertical space that it's hard to see them at the same time to understand what is being done.
Apart from that, great job 👏🏻

Reply:
I have updated the code. Does it match what you were requesting?

Reply:
Ah not really, I'm sorry that I was unclear. 

I wanted to split in two parts

- a part `<html><head>....</head></html>` with the associated Kotlin code
- a part `<html><body>...</body></html>` with the associated Kotlin code

This way we can look at both the HTML and the Kotlin code on the same screen without having to scroll back and forth

Reply:
Understood, that makes more sense. 

I have split the examples out a bit. However, in the interest of space (and maybe leaving it up as an exercise for the reader) I did not fully show what getting all the different elements/attributes in the HTML's `<body>` look like.

Reply:
That's fine, the samples are not intended as a complete reference,
the goal is to get people interested in the library,
and give them a starting point if they want to try out deeper themselves